src/core/cgroup.c

```diff
@@ -360,6 +361,14 @@ void cgroup_context_dump(CGroupContext *c, FILE* f, const char *prefix) {
                 (void) in_addr_to_string(iaai->family, &iaai->address, &k);
                 fprintf(f, "%sIPAddressDeny=%s/%u\n", prefix, strnull(k), iaai->prefixlen);
         }
+
+        STRV_FOREACH(path, c->ip_filters_ingress) {
+                fprintf(f, "%sIPFilterIngress=%s/\n", prefix, *path);
+        }
```

Comment:
no {} around single-line blocks...

Hmm, why the / suffix? are programs in bpffs exposed as directories?